<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmVenta</class>
 <widget class="QDialog" name="frmVenta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Venta</string>
  </property>
  <widget class="QLabel" name="lblTallas">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Escoja la talla de su camisa: </string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPagos">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Seleccione su método de pago:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblResultado">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;Muestra talla y método de pago&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>201</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vlyTalla">
    <item>
     <widget class="QRadioButton" name="rbtTallaM">
      <property name="text">
       <string>M</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbtTallaL">
      <property name="text">
       <string>L</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbtTallaXL">
      <property name="text">
       <string>XL</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbtTallaXXL">
      <property name="text">
       <string>XXL</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vlyPago">
    <item>
     <widget class="QRadioButton" name="rbtPagoTarjeta">
      <property name="text">
       <string>Tarjeta Débito/Crédito</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbtPagoContra">
      <property name="text">
       <string>Pago Contrareembolso</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbtPagoTransferencia">
      <property name="text">
       <string>Trasferencia Bancaria</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
